backend/src/v2/component/launcher_v2.go

```diff
@@ -628,7 +628,12 @@ func uploadOutputArtifacts(ctx context.Context, executorInput *pipelinespec.Exec
 			}
 
 			// Upload artifacts from local path to remote storages.
-			localDir, err := LocalPathForURI(outputArtifact.Uri)
+			localDir, err := retrieveArtifactPath(outputArtifact)
+
+			if outputArtifact.CustomPath != nil {
+				localDir = *outputArtifact.CustomPath
+			}
```

Comment:
Am I missing something, or does this "duplicate" the above statement?
It seems like it can be removed.